5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh

```diff
@@ -113,6 +113,10 @@ phpmyadmin_initialize() {
         replace_in_file "$PHPMYADMIN_CONF_FILE" "^(\s*//\s*)?(\\\$cfg\['Servers'\]\[\\\$i\]\['.*']\s*=)" "\2" true
     fi
 
+    # Configure allow order/rules settings
+    ! is_empty_value "$ALLOWDENY_ORDER" && info "Setting AllowDeny order option" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['AllowDeny']['order']" "$ALLOWDENY_ORDER"
+    ! is_empty_value "$ALLOWDENY_RULES" && info "Setting AllowDeny rules option" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['AllowDeny']['rules']" "$ALLOWDENY_RULES" yes
+
```

Comment:
Remove the info since we are not using it for other parameters:
```suggestion
    ! is_empty_value "$CONFIGURATION_ALLOWDENY_ORDER" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['AllowDeny']['order']" "$ALLOWDENY_ORDER"
    ! is_empty_value "$CONFIGURATION_ALLOWDENY_RULES" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['AllowDeny']['rules']" "array($ALLOWDENY_RULES)" yes
```

Apart from that, you should validate `CONFIGURATION_ALLOWDENY_ORDER` with something like this: 

```
   if [[ -n "$CONFIGURATION_ALLOWDENY_ORDER" ]]; then
        if ! [[ "$CONFIGURATION_ALLOWDENY_ORDER" =~ ^(deny,allow|allow,deny|explicit)$ ]]; then
            print_validation_error "The allowed values for CONFIGURATION_ALLOWDENY_ORDER are: 'deny,allow', 'allow,deny', or 'explicit'."
        fi
    fi
```